src/table/body/table-row.component.ts

```diff
@@ -218,4 +220,24 @@ export class TableRowComponent {
 	getExpandButtonAriaLabel(): Observable<string> {
 		return this._expandButtonAriaLabel.subject;
 	}
+
+	getItemTitle(item) {
+		if (!item || !item.data) {
+			return "";
+		}
+
+		if (typeof item.data === "string") {
+			return item.data;
+		}
+
+		if (item.data.title) {
+			return item.data.title;
```

Comment:
Is there somewhere we can document this? Also `item.data.title` vs. `item.title`? It may make more sense if we make it an optional property on the `TableItem` (for bonus points we could even have it proxy item.data when it's unset 🤔)

Reply:
Good idea!